app/views/content/life-as-a-teacher/pay-and-benefits/_what-teachers-have-to-say.html.erb

```diff
@@ -8,22 +8,19 @@
             <ul>
               <%= render Content::InspirationCardComponent.new(heading_tag: "h3", card:
```

Comment:
Do we need the ellipsis in line 4 (`Find out what teachers have to say...`)? We don't use them elsewhere and screen readers read them out which seems a bit verbose. But if there is a strong argument for keeping them, that's fine!

Reply:
I've updated this to remove them. Don't think there's any strong content argument to have them in and makes sense to remove anything that would make it more difficult for screen readers.